### PR TITLE
shalika- adding mock-server script in package.json

### PR DESCRIPTION
**Pull request by shalika10**

### CODE REVIEW

**Comment by shalika10** [comment]
@tiagostutz can you please verify the PR and merge it ,if it has no issue

**Comment by shalika10** [comment]
@tiagostutz any update ?

**Comment by tiagostutz** [comment]
Hi @shalika10 👋🏻 !
First of all, thanks for your PR! =)

I'm reviewing it right now and will come back asap.

File: app/package.json [hunk 1]
```diff
@@ -16,7 +16,10 @@
     "react-i18next": "^11.7.0",
     "react-router-dom": "^5.2.0",
     "react-scripts": "^3.4.3",
-    "typescript": "^3.8.0"
+    "typescript": "^3.8.0",
+    "body-parser": "^1.19.0",
+    "cors": "^2.8.5",
+    "express": "^4.17.1"
```
Comment (tiagostutz) [inline app/package.json:22]: I think there's no need for those dependencies (body-parser, cors and express) in this `package.json` as it is the frontend package.json.

Try to remove then and check whether the scripts keep working as supposed

**Review by tiagostutz — COMMENTED** [review]
Just submitted a review.

File: app/package.json (hunk 1 above)
Comment (shalika10) [reply app/package.json:22]: sure @tiagostutz   earlier- without these dependencies - the script was not working ,let me try that out again

Comment (shalika10) [reply app/package.json]: @tiagostutz any update ??

Comment (shalika10) [reply app/package.json]: hey @tiagostutz  i was tar-getting to close this in October it-self ,so it will be great if you can help me in closing this PR

Comment (tiagostutz) [reply app/package.json]: Sorry about late merge, @shalika10... =/

I just merged the PR.! Thanks a lot!

**Pull request merged**